configure.ac

```diff
@@ -637,6 +637,20 @@
         exit 1
     fi
 
+    PCRE2=""
+    #AC_CHECK_HEADER(pcre2.h,,[AC_MSG_ERROR(pcre2.h not found ...)])
```

Comment:
On my machine, this fails even if I have pcre2.h in my /usr/local/include directory

```
checking pcre.h usability... yes
checking pcre.h presence... yes
checking for pcre.h... yes
checking for pcre_get_substring in -lpcre... yes
checking pcre2.h usability... no
checking pcre2.h presence... no
checking for pcre2.h... no
configure: error: pcre2.h not found ...
```

Any idea why ? (it is a symbolic link unlike pcre.h)

Reply:
You `config.log` should hold some more info.

Reply:
Thanks, my PCRE2 version asks `PCRE2_CODE_UNIT_WIDTH` to be defined before including pcre2.h